src/front_input.c

```diff
@@ -1289,6 +1289,27 @@ short get_creature_control_action_inputs(void)
             }
             message_add(CrInst, get_string(StrID));
         }
+        if (is_key_pressed(KC_SEMICOLON,KMod_NONE))
```

Comment:
Could you make it a 'hold' instead of a 'toggle'?

Reply:
Having to hold the key can be annoying if you plan on doing a lot of claiming or reinforcing.

Reply:
The claiming I asked for it to be automatic, so then that would not be that bad. Also, with the key configurable, you can make it a key that suits you. I know you use arrow keys in possession, so you could rebound to 0. By default most people would use WASD and having Shift as a hold key (the default possession key) it's very easy to hold.

Reply:
Also, with the new key, clicking has become very difficult, I just have to really, really briefly touch the key or I toggle it too many times now.

Reply:
> 
> 
> Also, with the new key, clicking has become very difficult, I just have to really, really briefly touch the key or I toggle it too many times now.

That happened to me before with the semicolon, but I've no idea how to fix it :(.

Reply:
Well, making it a 'hold' rather than toggle should fix it anyway.